### PR TITLE
added read Gbp limit for graceful exit.  updated icon.

### PR DESCRIPTION
**Pull request by dcchivian**

### CODE REVIEW

File: lib/installed_clients/ReadsAPIClient.py [hunk 1]
```diff
@@ -25,7 +24,7 @@ def __init__(
             password=None, token=None, ignore_authrc=False,
             trust_all_ssl_certificates=False,
             auth_svc='https://ci.kbase.us/services/auth/api/legacy/KBase/Sessions/Login',
-            service_ver='release',
+            service_ver='beta',
```
Comment (scanon) [inline lib/installed_clients/ReadsAPIClient.py:27]: Do we want to do this?  Or is this just for testing?

**Review by scanon — COMMENTED** [review]
LGTM.  Just one comment/question.

File: lib/installed_clients/ReadsAPIClient.py (hunk 1 above)
Comment (bio-boris) [reply lib/installed_clients/ReadsAPIClient.py:27]: https://github.com/kbase/narrative/blob/develop/kbase-extension/static/kbase/js/widgets/function_output/kbaseReadsViewer.js uses it

Comment (dcchivian) [reply lib/installed_clients/ReadsAPIClient.py]: ok.  so we won't get rid of ReadsAPI.  it should get TLC then.  But that's beyond the scope of this module.  I removed the dependency on ReadsAPI for hipmer.

**Pull request merged**